docs/gds-supported-platform/resizing-persistent-volumes.md

```diff
@@ -0,0 +1,96 @@
+# Resizing persistent volumes
+
+The volumes you created in the beginning may not be big enough for
+your ongoing needs.
+
+As an example, we're going to double the space on the Concourse Worker
+persistent volumes - from 64Gi to 128Gi. Replace
+`gsp-concourse-worker` with the name of the resource you want to
+resize the volume for.
+
+1. Make a change to `charts/gsp-system/values.yml` with your desired
+   sizing, like [in PR 343](https://github.com/alphagov/gsp/pull/343):
+
+```yaml
+...
+persistence:
+  worker:
+    size: 128Gi
+```
+
+1. When merged, the PR will be released to sandbox. Things must pass
+   through sandbox via the full resize process to make sure we haven't
+   broken anything before going to the Verify cluster.
+
+1. Pods in StatefulSets each have a volume. For Kubernetes to be able
+   to resize the volumes, we need to detach them from the pods, by
+   destroying the stateful set. The `--cascade=false` flag is
+   important, as you don't want to delete the pods inside the stateful
+   set.
+
+   This step should be done as the Concourse release pipeline starts
+   to fail with an error message like:
+
+    > The StatefulSet “gsp-concourse-worker” is invalid: spec:
+    > Forbidden: updates to statefulset spec for fields other than
+    > ‘replicas’, ‘template’, and ‘updateStrategy’ are forbidden
+
+   To destroy the StatefulSet, run:
+
+    ```sh
+    $ gds sandbox kubectl -n gsp-system delete statefulset --cascade=false gsp-concourse-worker
+    ```
+
+   The `-deployer` pipeline will re-apply the statefulset. This will
+   intentionally roll all the associated pods.
+
+1. Once the pods are in a `Running` state, edit the
```

Comment:
How do I know the pods are in a `Running` state?

Reply:
Are you suggesting we include a command example here? As long as we don't have to explain all the options passed to it, shouldn't be too much of a digression.

Reply:
Depends on if we are confident the user knows how to check this? If you think we need to tell people this then I would include it e.g.
" Check that the pods are in a `Running`state: 
```
CODE
```"
However, if this isnt necessary, then you can keep it as is.

Reply:
I think the target audience for this document will know how to check. So I think I'll leave that for now.